SUMMARY.md

```diff
@@ -9,8 +9,9 @@
 - [🎓 Guides](guides/README.md)
   - [🚶 First Steps in Crypto](guides/newbie.md)
   - [💶 Get agEUR](guides/getAgEUR.md)
```

Comment:
I think I had broken the get agEUR reference (cf here: https://docs.angle.money/overview/guides/get-ageur), can you fix it pls?